packages/ckeditor5-mention/tests/manual/mention-custom-dropdownlimit.html

```diff
@@ -0,0 +1,9 @@
+<div id="editor">
+	<p>Hello <span class="mention" data-mention="@Ted">@Ted</span>.</p>
+	<p>Hello <span class="mention" data-mention="@Ted">#Ted</span></p>
```

Comment:
I think we should use the actual mentions available in this test (so `@01`, `@02` etc. for `@` and `#a01`, `#a02` etc. for `#`).